webapp/channels/src/components/user_settings/notifications/user_settings_notifications.tsx

```diff
@@ -1266,52 +1318,68 @@ class NotificationsTab extends React.PureComponent<Props, State> {
                         callsSelectedSound={this.state.callsNotificationSound || 'default'}
-                        isCollapsedThreadsEnabled={this.props.isCollapsedThreadsEnabled}
-                        areAllSectionsInactive={this.props.activeSection === ''}
                         isCallsRingingEnabled={this.props.isCallsRingingEnabled}
                     />
                     <div className='divider-light'/>
                     <EmailNotificationSetting
-                        activeSection={this.props.activeSection}
+                        active={this.props.activeSection === 'email'}
                         updateSection={this.handleUpdateSection}
-                        enableEmail={this.state.enableEmail === 'true'}
                         onSubmit={this.handleSubmit}
                         onCancel={this.handleCancel}
-                        onChange={this.handleEmailRadio}
                         saving={this.state.isSaving}
-                        serverError={this.state.serverError}
-                        isCollapsedThreadsEnabled={this.props.isCollapsedThreadsEnabled}
+                        error={this.state.serverError}
                         setParentState={this.setStateValue}
+                        areAllSectionsInactive={this.props.activeSection === ''}
+                        isCollapsedThreadsEnabled={this.props.isCollapsedThreadsEnabled}
+                        enableEmail={this.state.enableEmail === 'true'}
+                        onChange={this.handleEmailRadio}
                         threads={this.state.emailThreads || ''}
                     />
                     <div className='divider-light'/>
                     {pushNotificationSection}
                     <div className='divider-light'/>
                     {keywordsWithNotificationSection}
-                    <div className='divider-light'/>
-                    {keywordsWithHighlightSection}
+                    {!this.props.areFeaturesDisabled && (
+                        <>
+                            <div className='divider-light'/>
+                            {keywordsWithHighlightSection}
+                        </>
+                    )}
                     <div className='divider-light'/>
                     {!this.props.isCollapsedThreadsEnabled && (
                         <>
-                            {commentsSection}
                             <div className='divider-light'/>
+                            {commentsSection}
                         </>
                     )}
                     {this.props.enableAutoResponder && (
-                        autoResponderSection
+                        <>
+                            <div className='divider-light'/>
+                            {autoResponderSection}
+                        </>
+                    )}
+
+                    {/*  We placed the disabled items in the last */}
+                    {this.props.areFeaturesDisabled && (
```

Comment:
For the case of team edition, do we want to show disabled with the upsell or not show the section/upsell at all? I believe we've limited the upsells for enterprise editions E0 only, is that the case @abhijit-singh @johndavidlugtu ?